src/Explorer/Panes/__snapshots__/PublishNotebookPaneComponent.test.tsx.snap

```diff
@@ -14,7 +14,7 @@ exports[`PublishNotebookPaneComponent renders 1`] = `
   >
     <StackItem>
       <Text>
-        This notebook has your data. Please make sure you delete any sensitive data/output before publishing.
+        When published, this notebook will appear in the Azure Cosmos DB notebooks public gallery. Make sure you have removed any sensitive data or output before publishing.
```

Comment:
>data [](start = 140, length = 4)

sensitive data/output meant sensitive output but now with updated string, i am not sure,